apps/dashboard/app/views/layouts/application.html.erb

```diff
@@ -3,9 +3,17 @@
 <head>
   <title><%= content_for?(:title) ? yield(:title) : "Dashboard - #{OodAppkit.dashboard.title}" %></title>
   <%= favicon_link_tag 'favicon.ico', href: OodAppkit.public.url.join('favicon.ico'), skip_pipeline: true %>
-  <%= stylesheet_link_tag    'application', media: 'all' %>
+
+  <%= stylesheet_link_tag 'application', media: 'all' %>
+  <%= stylesheet_pack_tag 'application', media: 'all' %>
```

Comment:
I'm not sure I understand the addition of these two `stylesheet_link_tag` and `stylesheet_pack_tag` since we are using webpacker for only js right now